flowcraft/generator/templates/downsample_fastq.nf

```diff
@@ -17,7 +17,7 @@ process downsample_fastq_{{ pid }} {
     {% include "post.txt" ignore missing %}
 
     tag { "${sample_id}" }
-    publishDir "results/downsample_fastq_{{ pid }}/", pattern: "_ss.*"
+    publishDir "results/downsample_fastq_{{ pid }}/", pattern: "*_ss.*"
```

Comment:
Wait is this related with this PR? Or is it another fix?

Reply:
Ah it was a bug that I found that I commited to the wrong branch... Oh well :stuck_out_tongue:

Reply:
Added to changelog